pkgs/by-name/co/corrupter/package.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nix-update-script
+}:
+
+buildGoModule {
+  pname = "corrupter";
+  version = "1.0";
```

```suggestion
  version = "1.0-unstable-YYYY-MM-DD";
```
need to add the `-unstable-YYYY-MM-DD` tag to version as noted below. the date is from the revision used.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming
> If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.